### DISCUSSION TITLE
Configuration cache cannot clear when deployment

### DISCUSSION THREAD

**Discussion by kai0310** [spatie/laravel-settings#101] [Q&A] [answered]
## The Problem 👀
I did not have this problem in the development environment, but I had the following problem when deploying.
The following error message is displayed when executing the following command: `php artisan config:cache`

```

   LogicException 

  Your configuration files are not serializable.

  at vendor/laravel/framework/src/Illuminate/Foundation/Console/ConfigCacheCommand.php:71
     67▕             require $configPath;
     68▕         } catch (Throwable $e) {
     69▕             $this->files->delete($configPath);
     70▕ 
  ➜  71▕             throw new LogicException('Your configuration files are not serializable.', 0, $e);
     72▕         }
     73▕ 
     74▕         $this->info('Configuration cached successfully!');
     75▕     }

  1   bootstrap/cache/config.php:538
      Error::("Call to undefined method Closure::__set_state()")

      +14 vendor frames 
  16  artisan:37
      Illuminate\Foundation\Console\Kernel::handle()
```

## Supplement 📝
I referred to [this article](https://stackoverflow.com/questions/52065513/your-configuration-files-are-not-serializable), but it did not solve the problem.

## Environment 📦
- Laravel 8.55
- spatie/laravel-settings 2.2

**Comment by rubenvanassche** [comment, marked as answer]
Hi @kai0310,

I don't see how this could be a problem of the settings package, the `config:cache` command will cache all the config files within your Laravel application. The `settings.php` config file only contains strings and no closures or objects so it can be serialized without problems.

**Reply by kai0310** [reply]
This problem was not caused by the package. Thanks :)
